src/state/reducers/articleReducer.js

```diff
@@ -0,0 +1,17 @@
+import { GET_ALL_ARTICLES } from '../actions/types';
+
+const initialState = {
+  articles: []
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case GET_ALL_ARTICLES:
+      return {
+        ...state,
+        articles: action.payload
+      };
+    default:
+      return state;
+  }
+};
```

Comment:
Inside state, create an article folder. Inside article folder, you create actions, reducers, initial state, and an index file.